generated_files

```diff
@@ -9,6 +9,9 @@ mixer/adapter/kubernetes/config/config.pb.go
 mixer/adapter/list/config/config.pb.go
```

Comment:
Please merge with most recent code change.  Since the renaming, the CRD defintion in https://github.com/istio/istio/tree/master/install/kubernetes must be updated as well, including 
istio*.yaml and their templates in install/kubernetes/templates.

Reply:
PTAL.